frontend/src/components/QueueTable.js

```diff
@@ -48,6 +54,7 @@ function QueueTable({ queueData: { name, rows } }) {
       </IconButton>
     )
   }
+
   const SeenButton = () => {
     return (
       <IconButton>
```

Comment:
@hhummel I dont see where these things are being updated.  Shouldnt changes in status, notes, seen and urgency all have an update call made to the database?  Like calling updateQueue?  I dont see where this is happening.  My thought is that it would happen in the component, `QueueTableDropdown` but I dont see it there.  I see a handle change event in that file, but its just changing a state param in the component, and not using the Queue store at all.  Perhaps this needs to be added.  One question I have is if you think this is too much work to add to this pr and if we should ticket it out to another just to get this on master.

Reply:
I update the component on render, but not other changes.  Those weren't in the original ticket

Reply:
I think we should pull, and then add the other changes. Marcel's PR is based on the old AllQueues, for example

Reply:
ok, I made a new ticket for this